guidelines/index.html

```diff
@@ -389,7 +389,7 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
             		
             
             
-            <p>WCAG 2.2 meets a set of <a href="https://w3c.github.io/wcag/requirements/22/">requirements for WCAG 2.2</a> which, in turn, inherit requirements from previous WCAG 2 versions. 要件は、ガイドラインの全体的なフレームワークを構成し、後方互換性を確保している。ワーキンググループはまた、達成基準のスタイルや品質が WCAG 2.0 のものと類似することを確保するために、達成基準に対し、あまり形式的ではない一連の受け入れ基準を採用した。これらの要件は、WCAG 2.2に含めることができるものを制限している。この制限は、WCAG 2 のドットリリースの性質を保持するために重要である。</p>
+            <p>WCAG 2.2 は、過去の WCAG 2 バージョン群の要件を継承する一連の <a href="https://w3c.github.io/wcag/requirements/22/">WCAG 2.2 の要件</a>を満たしている。要件は、ガイドラインの全体的なフレームワークを構成し、後方互換性を確保している。ワーキンググループはまた、達成基準のスタイルや品質が WCAG 2.0 のものと類似することを確保するために、達成基準に対し、あまり形式的ではない一連の受け入れ基準を採用した。これらの要件は、WCAG 2.2に含めることができるものを制限している。この制限は、WCAG 2 のドットリリースの性質を保持するために重要である。</p>
```

Comment:
リンクは原文に戻しておくとして、以下はいかがでしょうか

```suggestion
            <p>WCAG 2.2 は、以前の WCAG 2 のバージョンから要件を継承する一連の <a href="https://w3c.github.io/wcag/requirements/22/">requirements for WCAG 2.2</a> を満たしている。要件は、ガイドラインの全体的なフレームワークを構成し、後方互換性を確保している。ワーキンググループはまた、達成基準のスタイルや品質が WCAG 2.0 のものと類似することを確保するために、達成基準に対し、あまり形式的ではない一連の受け入れ基準を採用した。これらの要件は、WCAG 2.2 に含めることができるものを制限している。この制限は、WCAG 2 のドットリリースの性質を保持するために重要である。</p>
```

Reply:
なるほど、「以前の」とすればすっきりしますね。イイと思います！